### ISSUE TITLE
Remove/Rename ibctesting.FirstChannelID

### ISSUE THREAD

**Issue by bznein** [cosmos/ibc-go#6765]
`ibctesting.FirstChannelID` used to be a valid constant to represent channels used in testing. However, after #6748, this is not true anymore.

We should revisit all occurrences of this constant and either:
- Replace them with the appropriate channel 
- Replace them with a properly named constant (most time they can either represent invalid channels or placeholders)

**Comment by srdtrk** [comment]
Please do not remove this constant from the ibctesting package as I'm using it in external crates. Maybe the issue should be renamed to `Remove/Revisit the usage of ibctesting.FirstChannelID` since the constant is appropriately named?

**Comment by chatton** [comment]
Yeah I think keeping the constant in place is doing no harm to not mess with external dependencies on it. As long as our tests are updated to not rely on it, I think it's fine to keep.

**Comment by gjermundgaraba** [comment]
I see no reason to make any changes here right now, so will close this for now.
